### DISCUSSION TITLE
part-ii/2006-23/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#220] [Question comments]
Part II 2006 Coding and Cryptography: 4.I.4G 

https://questions.tripos.org/part-ii/2006-23/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

Theorem: (Shannon's second coding theorem) For a discrete memoryless channel, the operational capacity is equal to the information capacity.

For the capacity of a binary erasure channel, take input $X$ with $\mathbb{P}(X=0) = \alpha$, $\mathbb{P}(X=1) = 1 - \alpha$. We have output $Y$ with $\mathbb{P}(Y=0) = \alpha(1-p)$, $\mathbb{P}(Y=1) = (1-\alpha)(1-p)$, $\mathbb{P}(Y=*) = p$.

Then by Shannon's second coding theorem, we can compute the information capacity, which is equal to 

$$
\begin{align*}
C &= \max I(X,Y) \\
&= \max_{\alpha} H(X) - H(X|Y) \\
&= \max_{\alpha} H(\alpha) - pH(\alpha) \\
&= (1 - p) \max_{\alpha} H(\alpha) \\
& = 1 - p \\
\end{align*}
$$

So the capacity of the binary erasure channel is $1 - p$.

</details>
